test/utils.py

```diff
@@ -565,6 +567,59 @@ def is_tarantool_ee():
     return False
 
 
+def skip_if_tarantool_ce():
+    if not is_tarantool_ee():
+        pytest.skip("Tarantool Enterprise required")
+
+
+def is_quit_supported():
+    major, minor = get_tarantool_version()
+    return major >= 2
+
+
+def is_cluster_app_supported():
+    major, minor = get_tarantool_version()
+    return major >= 3
+
+
+def is_tuple_format_supported():
+    major, minor = get_tarantool_version()
+    return major > 3 or (major == 3 and minor >= 2)
+
+
+def is_tarantool_major_one():
+    cmd = ["tarantool", "--version"]
+    instance_process = subprocess.run(
+        cmd,
+        stderr=subprocess.STDOUT,
+        stdout=subprocess.PIPE,
+        text=True
+    )
+    if instance_process.returncode == 0:
+        return "Tarantool 1." in instance_process.stdout
+    return False
```

Comment:
Why just don't use `get_tarantool_version`?
```suggestion
   major, minor = get_tarantool_version()
    return major == 1
```

Reply:
Yeap, we can. Updated.